ramls/loan.json

```diff
@@ -112,6 +112,27 @@
       "description": "Metadata about creation and changes to loan, provided by the server (client should not provide)",
       "type": "object",
       "$ref": "raml-util/schemas/metadata.schema"
+    },
+    "claimedReturned": {
+      "type": "object",
+      "description": "Loan claimed returned",
+      "properties": {
+        "dateTime": {
+          "description": "Date and time the item was claimed returned for this loan",
+          "type": "string",
+          "format": "date-time"
+        },
+        "staffMemberId": {
+          "type": "string",
+          "description": "Staff member id who marked the loan as claimed returned",
+          "$ref": "raml-util/schemas/uuid.schema"
+        }
```

Comment:
Why does the staff member who marked an item as claimed returned during this loan need to be recorded? Is there a scenario in the dependant stories that requires this?

Reply:
Yes, check scenario # 6:
> Given a loan for an item that is Claim returned
> * When viewing loan details
> * Then populate as follows: 
>    * Claimed returned: show date and time item was claimed returned
>    * Action history: show row for claimed returned
>      * ...
>      * Source: Lastname, firstname (and link to user profile) of user who marked item claim returned

Reply:
@bohdan-suprun Thanks

As I understand it, the action history is based upon the audit tracking of the loans table in mod-circulation-storage. 

I think `Source` typically refers to the change metadata included in each record. Is that sufficient for this change, or are we expecting the UI to include special logic for the `claimed returned` action to use different information from the record?

Does that fit with your understanding?

Reply:
Thanks, @marcjohnson-kint. That's make sense. I think we have to check with Emma, how important the staff member id for this feature. Because it maybe a case when:
* User `A` marks the item as claimed returned: `updatedByUserId` is `A`
* Then user `B` updates/closes the loan, so the `updatedByUserId` is `B` now.

What do you think?

Reply:
@bohdan-suprun I think checking with Emma is ok.

If I'm understanding scenario 6 correctly, this information is only relevant for action history, which will not be effected by subsequent changes to the loan because it is based on copied snapshots of the loan on every write.

Does that make sense?

Reply:
Aha, right. Will remove the `staffMemberId` then.